.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,8 +2,17 @@ Fixes #
 
 To test:
 
+Regression Notes
```

Comment:
If we're re-working this, I wonder if we should make this and "To test" headers?

```suggestion
## Regression Notes
```